frontend/src/index.css

```diff
@@ -0,0 +1,65 @@
+/* NAVBAR  */
```

Comment:
Nah, get all this in some styled-components. Maybe not this second, but before we're done.